src/main/java/org/jabref/model/entry/AuthorList.java

```diff
@@ -286,6 +272,15 @@ public Author getAuthor(int i) {
         return authors;
     }
 
+    public List<Author> getLatexFreeAuthors() {
```

Comment:
One could use the same strategy as for the Author class here, and introduce a `AuthorList latexFree()` method. Then one could also remove all the "latexFree" versions of the methods below.

Reply:
I'll try to `implements List<Author>` with a cache strategy so that repeated `AuthorList.latexFree().latexFree()...` calls returns the same `AuthorList` (I am trying to make it immutable anyway).

Reply:
The repeated calls you might be able to deal with as follows:
```
latexFree() {
  if authorsLatexFree == null
      authorsLatexFree = AuthorList.of(unicode converted...)
      authorsLatexFree.authorsLatexFree = authorsLatexFree 
}
return authorsLatexFree;
```
but honestly, I wouldn't worry too much about it. In the normal usage, you get the author list from an entry and then use `latexFree` once if you want to have a latexfree version.

Implementing `List<Author>`  is a good idea but might be a lot of work. For example, you want to make sure that add/remove etc update also the latexfree cache. Maybe it's enough for now to implement `AuthorList.stream()`.

Reply:
> Implementing `List<Author>` is a good idea but might be a lot of work

I have to admit I have not looked into it in too much detail yet. My "high-level" approach were to make an unmodifiable list based on `util.AbstractList`. At least the API documentation claims it shouldn't be too hard 😜 

#### Regarding immutability
I believe `AuthorList` should be unmodifiable because,
1. It is non-trivial to add/remove authors while dealing with bibtex. `{Barnes and Noble} and Author` would need to insert Noble between Barnes and Author if the braces are removed. It is much easier to just re-parse it.
2. There is currently no parts of JabRef that relies on a list of authors being immutable. Barring AuthorListParser I have replaced all uses with Streams. The mutability required by `AuthorListParser` can be addressed but I haven't decided what is the best solution, such that it does not come at a performance cost. (I am leaning towards making it an inner class of `AuthorList` to avoid creating an extra list/array through `AuthorList.of`)
3. Even if I am wrong, all static methods in `AuthorList` can (and in my opinion should) be made to work with `List<Author>`. Worst case scenario, someone that must have `AuthorList` mutability can use any class extending `List` and use static calls instead.
 
#### Regarding mutability
Because I think immutability is the right approach, I haven't really spent as much thought on this option. I think that using an `EnumMap` (or anything with an iterator) to store cached lists of `AuthorList`s that are extended by Decorators would allow an easy and less error prone implementation. Adding/removing elements can be overridden to primarily work with indices.

E.g., if a latex free `AuthorList` wants to remove a specific `Author`, first make sure that the `Author` is latex free and then find the index of that `Author` (if it is in the list), then use an iterator over the cached `AuthorList`s to remove that specific index. It is really not safe for concurrent modifications but neither is the current one.

I can add a "draft" implementation of one of these decorators if you'd prefer that to this explanation.

"P.S.", if you think mutability is the way to go, feel free to tell me that, and I'll look into it more.

Reply:
My first thought would be to have it as a mutable list. This might become handy in the future, although I don't really have a use case right now. But for now I would keep the structure like it is right now with a wrapped `List<Author>` variable. Of course, if you want to add convient methods like `stream()` or something like this, you are very welcome to do so. I suggest we keep it growing organically, and if at some point many of the list methods are implemented we can also fully implement the interface.

Reply:
Ok. In that case I'll leave it (mostly) as is. I'll add the most basic variant of `public AuthorList latexFree()` and remove all latex free methods.

I don't want to add a stream utility method without implementing an unmodifiable list. I can't sort it out in my head when I'd want a method to return a list (getAuthors) and when it should return an AuthorList and what the meaningful distinction between the two should mean for a stream. That probably means I'd make a poor job of it anyway 😛

Reply:
In general, one should always use `AuthorList` over manually handling `List<Author>`. The only class that should have `List<Author>` in it should be `AuthorList`.

For the stream method, I simply thought about having a `Stream stream() {return authors.stream();}` method. But maybe it's a good idea to not expose this for now, since otherwise people will implement some form of `AuthorList -> str` methods in various places, and the purpose of the `AuthorList` class was to combine these things anyway.

Reply:
> `AuthorList -> str` methods in various places

The `AuthorList -> ?` is needed in various places. Take institute authors, both in
https://github.com/JabRef/jabref/blob/54f27e94bc1108f5c6cb485bb2daac938f0ec573/src/main/java/org/jabref/logic/msbib/MSBibConverter.java#L127-L131

and

https://github.com/JabRef/jabref/blob/54f27e94bc1108f5c6cb485bb2daac938f0ec573/src/main/java/org/jabref/logic/citationkeypattern/BracketedPattern.java#L497-L501

In both locations it would be better if they rely on the parsing of `AuthorListParser` and only utilize `List<Author>`.
I even have a `isInstitute()` method in one of my local `Author` variants ^^

You can also view #7228 as related to this.

Reply:
I dunno. Do you think this discussion will lead to anything concrete?
I am happy just implementing `public AuthorList latexFree()` and killing of quite a few methods 😄 

As you might have noticed I have ideas both for dealing with `getLatexFreeField` and implementing `List`. Perhaps that is again better for a different PR and I can expand on the idea(s) so they can be criticised in a more structure fashion.

Reply:
Yes, these workarounds should actually be moved to the `AuthorListParser`. There was a dark time in JabRef's history, where there was no Author class and everyone reimplemented different parsing and serialization schemes again and again. Happily it's now 2021 ;-)

Reply:
I guess it makes sense to add the "Workarounds" for the Authorlist in one central place. And especially institution handling is important for exporters

Reply:
I'll add a follow-up PR moving the formatting code of `layout/Authors.java` to `AuthorList`. I believe they produce different results when using oxford commas for two authors, which is not ideal.